metricflow/plan_conversion/to_sql_plan/dataflow_to_subquery.py

```diff
@@ -793,17 +793,45 @@ def visit_order_by_limit_node(self, node: OrderByLimitNode) -> SqlDataSet:  # no
         )
 
     def visit_write_to_result_data_table_node(self, node: WriteToResultDataTableNode) -> SqlDataSet:  # noqa: D102
-        # Returning the parent-node SQL as an approximation since you can't write to a data_table via SQL.
-        return self.get_output_data_set(node.parent_node)
+        # Currently, this is a SELECT statement that passes all columns through. This will be updated in a later PR
+        # to support specific column ordering.
+        input_data_set: SqlDataSet = self.get_output_data_set(node.parent_node)
+        input_data_set_alias = self._next_unique_table_alias()
+        output_instance_set = input_data_set.instance_set.transform(
+            ChangeAssociatedColumns(self._column_association_resolver)
```

Comment:
This transformer doesn't do anything, right? We should probably delete it. I think you can just leave the instances as is.

Reply:
Good catch - I'll remove.